### PR TITLE
Added Digest Swarm Chain

### PR DESCRIPTION
**Pull request by quetzalcroix**
This PR adds the details for the Digest Swarm Chain to chainlist. Digest Swarm Chain is an EVM chain with social trade features for society group.

### CODE REVIEW

File: _data/icons/swarmexplorer.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+	"url":"https://ipfs.moralis.io:2053/ipfs/Qmbzy8c54EHh7yzMHZir98BrcV6sazr77QLoi63HAoayuo",
```
Comment (ligi) [inline _data/icons/swarmexplorer.json:3]:
```suggestion
	"url":"ipfs://Qmbzy8c54EHh7yzMHZir98BrcV6sazr77QLoi63HAoayuo",
```

File: _data/icons/swarmchain.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+	"url":"https://ipfs.moralis.io:2053/ipfs/QmRu7KmQoC2vSVeD7Y6bxRYikQHYs86H9fRAdb1ytPkK9r",
```
Comment (ligi) [inline _data/icons/swarmchain.json:3]:
```suggestion
	"url":"ipfs://QmRu7KmQoC2vSVeD7Y6bxRYikQHYs86H9fRAdb1ytPkK9r",
```

**Review by ligi — APPROVED** [review]

File: _data/icons/swarmexplorer.json (hunk 1 above)
Comment (quetzalcroix) [reply _data/icons/swarmexplorer.json]: Got it, gonna correct it ASAP, thanks

**Review by quetzalcroix — COMMENTED** [review]
Correction to IPFS URL

**Pull request merged**